static/js/compose.js

```diff
@@ -425,6 +425,9 @@ export function initialize() {
     $("#compose-textarea").on("input propertychange", () => {
         compose_validate.warn_if_topic_resolved(false);
         const compose_text_length = compose_validate.check_overflow_text();
+        if (compose_text_length === 0 && $("#compose-textarea").hasClass("invalid")) {
```

Comment:
I think the bug here is that this check for the size of the compose box is inverted.

Reply:
Wow I wonder at what point I inverted this -- it was definitely working for me when I was testing it at some point 🙃 thanks!

Reply:
Fixed!